ledger/applications.go

```diff
@@ -235,3 +238,15 @@ func (al *logicLedger) DelGlobal(key string) error {
 func (al *logicLedger) GetDelta(txn *transactions.Transaction) (evalDelta basics.EvalDelta, err error) {
 	return al.cow.BuildEvalDelta(al.aidx, txn)
 }
+
+func (al *logicLedger) AppendLog(txn *transactions.Transaction, value string) error {
+	idx, err := txn.IndexByID(txn.ApplicationID)
+	if err != nil {
+		return err
+	}
+	return al.cow.AppendLog(idx, value)
+}
+
+func (al *logicLedger) GetLogs() []basics.LogItem {
+	return al.cow.getLogs()
+}
```

Comment:
I do not see a need for both `GetLogs ` and `getLogs` - please remove. In `logic` tests there is a mock ledger instance available with direct access to logs.